hudi-common/src/main/java/org/apache/hudi/common/util/StringUtils.java

```diff
@@ -67,4 +69,29 @@ public static String toHexString(byte[] bytes) {
   public static boolean isNullOrEmpty(String str) {
     return str == null || str.length() == 0;
   }
+
+
+  /**
+   * Returns the given string if it is non-null; the empty string otherwise.
+   *
+   * @param string the string to test and possibly return
+   * @return {@code string} itself if it is non-null; {@code ""} if it is null
+   */
+  public static String nullToEmpty(@Nullable String string) {
+    return string == null ? "" : string;
+  }
+
+  /**
+   * Returns the given string if it is nonempty; {@code null} otherwise.
+   *
+   * @param string the string to test and possibly return
+   * @return {@code string} itself if it is nonempty; {@code null} if it is empty or null
+   */
+  public static @Nullable String emptyToNull(@Nullable String string) {
+    return stringIsNullOrEmpty(string) ? null : string;
+  }
+
+  private static boolean stringIsNullOrEmpty(@Nullable String string) {
+    return string == null || string.isEmpty();
+  }
```

Comment:
hi smarthi, thanks for opening the PR, I am wondering if we really need these methods? or will them be used somewhere later?

Reply:
yes these can be used as replacement for Guava-Strings in SparkMain.java.
Also see https://github.com/apache/incubator-hudi/pull/1159#discussion_r363821619 for the need to write test cases. 

The new methods will potentially be used in the bigger PR# 1159.